app/components/Texting/Texting.scss

```diff
@@ -0,0 +1,178 @@
+@import '../../styles/utils/_helpers.scss';
+
+.overlay {
+  position: fixed;
+  height: 100%;
+  width: 100%;
+  top: 70px;
+  background: rgba(0, 0, 0, 0.2);
+  @media screen and (max-width: 900px ){
+    top: 50px;
+  }
+}
+
+.popup {
+  position: relative;
+  width: 500px;
+  height: 660px;
+  background: #FFFFFF;
+  margin: 35px auto;
+  padding: 35px;
+  box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
+  border-radius: 8px;
+
+  @media screen and (max-width: 900px ) {
+    width: 100%;
+    height: 100%;
+    font-size: 14px;
+    padding: 20px;
+    margin: 0 auto;
+  }
+}
+
+.title{
+  font-style: normal;
+  font-weight: bold;
+  font-size: 1.8em;
+  line-height: 35px;
+}
+
+.description {
+  font-size: 1.3;
+  margin-top: 2em;
+}
+
+.input {
+  background-color: $color-white;
+  height: 50px;
+  @media screen and (max-width: 900px ) {
+    height: 35px;
+  }
+}
+
+.inputField{
+  margin-top: 2em;
+  @media screen and (max-width: 900px ) {
+    margin-top: 1em;
+  }
+}
+.label {
+  font-size: 1em;
+  color: $color-grey5;
+  margin-top: 2em;
+  display: block;
+}
+
+.dataRates {
+  font-size: 0.9em;
+  line-height: 30px;
+  color: $color-grey5;
+}
+.checkBox {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+  color: $color-grey5;
+  padding: 8px 0;
+  cursor: pointer;
+}
+
+.checkBox input[type="checkbox"] {
+  width: auto;
+  margin-right: 10px;
+  margin-left: 0;
+  cursor: pointer;
+}
+
+.button {
+  font-size: calc-em(16px);
+  padding: calc-em(7px) calc-em(15px);
+  background: $color-brand;
+  color: $color-white;
+  border-radius: 3px;
+  width: 30%;
+  margin-left: 20px;
+
+  @media screen and (max-width: 900px ) {
+    width: 100%;
+    font-size: 14px;
+    margin: 8px auto;
+  }
+  &.disabled {
+    background: $color-brand-light;
+  }
+  &.cancel {
+    border: 1px solid $color-brand;
+    color: $color-brand;
+    background: $color-white;
+    @media screen and (max-width: 900px ) {
+      display: none;
+    }
+    &:hover {
+      background: $color-brand;
+      color: $color-white;
+    }
+  }
+}
+
+.buttons {
+  display: flex;
+  margin-top: 2em;
+  justify-content: flex-end;
+  @media screen and (max-width: 900px ) {
+    margin-top: 0;
+  }
+}
+.privacy {
+  margin-top: 2em;
+  font-size: 14px;
+  text-align: center;
+  @media screen and (max-width: 900px ) {
+    margin-top: 1em;
+    font-size: 12px;
+  }
+}
+
+.closeIcon {
+  width: 17.42px;
+}
+
+.closeIconDiv {
+  display: flex;
+  justify-content: flex-end;
+  margin-bottom: 20px;
+  cursor: pointer;
+}
+
+.circle {
+    width: 150px;
+    height: 150px;
+    background: #00916A;
+    border-radius: 50%;
+    position: absolute;
```

Comment:
What is this `position: absolute` meant to do? It doesn't look like it has any effect on the positioning of this element, since when I disable it, it doesn't seem to change the page at all for me.